### PR TITLE
Remove unnecessary space for provided plugins, fixes #4608

### LINKED ISSUE #4608: Double space in front of provides scoped plugins
### What behaviour is expected:
No double space in front of the provided braces.

### What behaviour is observed:
When using the `provides` scope in your plugin.yml, Paper adds an extra space in front of the provided braces additional to the regular delimiter.

### Steps/models to reproduce:
1. Write a plugin that uses the provides scope
2. ...
3. See the difference

### Plugin list:
Same as the image above.

### Paper version:
This server is running Paper version git-Paper-211 (MC: 1.16.3) (Implementing API version 1.16.3-R0.1-SNAPSHOT)

### Anything else:
/

### PR DESCRIPTION
**Pull request by aurorasmiles**
Since there is already a Space after the , and the :, there is no need for an aditional space in front of the provided plugin list
Fixes #4608

### CODE REVIEW

**Review by NotMyFault — DISMISSED** [review]
Thx auwowa

File: Spigot-API-Patches/0228-Remove-unnecessary-space-for-provided-plugins.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+diff --git a/src/main/java/org/bukkit/command/defaults/PluginsCommand.java b/src/main/java/org/bukkit/command/defaults/PluginsCommand.java
+index fd74512a2bbbb0f198192b450e09bdbb9bac248d..3afa8f36929db4892796700abd837dbf4ff1608f 100644
+--- a/src/main/java/org/bukkit/command/defaults/PluginsCommand.java
++++ b/src/main/java/org/bukkit/command/defaults/PluginsCommand.java
+@@ -12,7 +12,7 @@ import org.bukkit.command.CommandSender;
+ import org.bukkit.plugin.Plugin;
+ import org.jetbrains.annotations.NotNull;
+ 
+-public class PluginsCommand extends BukkitCommand {
++public class  PluginsCommand extends BukkitCommand {
```
Comment (Proximyst) [inline Spigot-API-Patches/0228-Remove-unnecessary-space-for-provided-plugins.patch]: Moving the unnecessary space*

**Review by Proximyst — DISMISSED** [review]

**Review by kashike — DISMISSED** [review]

**Comment by Proximyst** [comment]
@limbo-app ready

**Review by limbo-app — APPROVED** [review]
Pull request automatically approved for merging - a core team member has previously approved this pull request.

**Pull request merged**